mqtt/src/test/java/akka/stream/alpakka/mqtt/javadsl/MqttSourceTest.java

```diff
@@ -213,8 +213,8 @@ public void receiveFromMultipleTopics() throws Exception {
 
   @Test
   public void supportWillMessage() throws Exception {
-    String topic1 = "source-spec/topic1";
-    String willTopic = "source-spec/will";
+    String topic1 = "source-test/topic1";
+    String willTopic = "source-test/will";
```

Comment:
Good! I remember doing this to other Java tests some time ago. We need to keep an eye on this when adding new MQTT tests.